testing/tests/ui/filter-signature-validation.stderr

```diff
@@ -0,0 +1,77 @@
+error: Filter function missing required input and environment args
+ --> tests/ui/filter-signature-validation.rs:3:19
+  |
+3 |     pub fn filter0() -> askama::Result<String> {}
+  |                   ^
+
+error: Filter function missing required input and environment args
+ --> tests/ui/filter-signature-validation.rs:6:19
+  |
+6 |     pub fn filter1(input: usize) -> askama::Result<String> {}
+  |                   ^
+
+error: Filter function missing required input and environment args
+ --> tests/ui/filter-signature-validation.rs:9:19
+  |
+9 |     pub fn filter2(_: &dyn askama::Values) -> askama::Result<String> {}
+  |                   ^
+
+error: Lifetime generics are currently not supported for filters
```

Comment:
Is there a reason for this limitation or it's just something to be implemented after this PR?